service/src/main/java/com/shortlink/service/server/RandomLtS.java

```diff
@@ -15,7 +15,7 @@ private char randomChar(){
 
     // Encodes a URL to a shortened URL.
     public String encode(String longUrl) {
-        String shortUrl = "http://";
+        String shortUrl = "http://localhost:7000/stl/";
```

Comment:
Same comment as above

Reply:
the endpoint is stored in application.properties